Fix some stuff in the man page. (Should be better sent to upstream. :-)

Index: autossh/autossh.1
===================================================================
--- autossh.orig/autossh.1	2011-11-19 02:34:15.000000000 +0100
+++ autossh/autossh.1	2011-11-19 02:57:51.000000000 +0100
@@ -60,7 +60,6 @@
 .Nm
 tries to distinguish the manner of death of the ssh process it
 is monitoring and act appropriately. The rules are:
-.Bl -tag -width Ds
 .It 1.
 If the ssh process exited normally (for example, someone typed
 "exit" in an interactive session), 
@@ -126,14 +125,14 @@
 poll time (usually 10 minutes).
 .Nm
 can be "prodded" to retry by signalling it, perhaps with
-SIGHUP ("kill -HUP").
+SIGHUP ("kill \-HUP").
 .Pp
 .Ss Connection setup
 .Pp
 As connections must be established unattended, the use of 
 .Nm
 requires that some form of automatic authentication be set up. The use
-of RSAAuthentication with ssh-agent is the recommended method. The
+of RSAAuthentication with ssh\-agent is the recommended method. The
 example wrapper script attempts to check if there is an agent running
 for the current environment, and to start one if there isn't.
 .Pp
@@ -160,8 +159,8 @@
 using. 
 .Nm 
 will send test data on the base monitoring port, and
-receive it back on the port above. For example, if you specify "-M
-20000", 
+receive it back on the port above. For example, if you specify
+.Dq Fl M 20000 ,
 .Nm
 will set up forwards so that it can send data on port
 20000 and receive it back on 20001.
@@ -241,12 +240,14 @@
 .It Ev AUTOSSH_MAXSTART
 Specifies how many times ssh should be started. A negative number
 means no limit on the number of times ssh is started. The default
-value is -1.
+value is \-1.
 .It Ev AUTOSSH_MESSAGE
 Append message to echo message sent when testing connections.
 .It Ev AUTOSSH_NTSERVICE 
 (Cygwin only.) When set to "yes" , autossh sets up to run as an NT
-service under cygrunsrv. This adds the -N flag for ssh if not already
+service under cygrunsrv. This adds the
+.Fl N
+flag for ssh if not already
 set, sets the log output to stdout, and changes the behaviour on ssh
 exit so that it will restart even on a normal exit.
 .It Ev AUTOSSH_PATH
@@ -261,15 +262,18 @@
 poll time.
 .It Ev AUTOSSH_PORT
 Sets the connection monitoring port. Mostly in case ssh appropriates
--M at some time. But because of this possible use, AUTOSSH_PORT
-overrides the -M flag. A value of 0 turns the monitoring function off.
+.Fl M
+at some time. But because of this possible use, AUTOSSH_PORT
+overrides the
+.Fl M
+flag. A value of 0 turns the monitoring function off.
 .El
 .Sh AUTHOR
 .Nm
 was written by Carson Harding.
 .Sh SEE ALSO
 .Xr ssh 1 ,
-.Xr ssh-add 1 ,
-.Xr ssh-agent 1 ,
-.Xr ssh-keygen 1 ,
+.Xr ssh\-add 1 ,
+.Xr ssh\-agent 1 ,
+.Xr ssh\-keygen 1 ,
 .Xr cygrunsrv 1 .
